docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -249,29 +286,42 @@ Lastly, we will add our update function:
 23 }
 ```
 
-The name `update` is arbitrary, but makes sense for our example. Notice that we use the `@method` decorator because we intend for this method to be invoked by end users, such as via a zkApp UI or our `main.ts` script in this case.
+The function name `update` is arbitrary, but it makes sense for this example. Notice how the `@method` decorator is used because it is intended to be invoked by end users by using a zkApp UI, or as in this case, the `main.ts` script.
+
+This method contains the logic by which end users are allowed to update the zkApp's account state on chain. In this example, the code specifies:
```

Comment:
oh, this is our first mention of

> zkApp's account state on chain

what is a zkApp account? and let's describe it here

Reply:
A zkApp account is an account is on the Mina blockchain were a zkApp smart contract is deployed. A zkapp account has a verification key associated with it. https://github.com/o1-labs/docs2/blob/16e813c318ff05e2101d3ff5a581d39bf6724aab/docs/zkapps/how-zkapps-work.mdx?plain=1#L90-L91